src/recipe/Baked.Recipe.Service.Application/CodingStyle/RichTransient/LookUpTransientsByIdsConvention.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Baked.Business;
+using Baked.Domain.Model;
+using Baked.RestApi.Configuration;
+using Humanizer;
+
+namespace Baked.CodingStyle.RichTransient;
+
+public class LookUpTransientsByIdsConvention(DomainModel _domain)
+    : IApiModelConvention<ParameterModelContext>
+{
+    public void Apply(ParameterModelContext context)
+    {
+        if (context.Action.MappedMethod is null) { return; }
+        if (context.Action.MappedMethod.Has<InitializerAttribute>()) { return; }
+        if (!context.Parameter.IsInvokeMethodParameter) { return; }
+        if (context.Parameter.MappedParameter is null) { return; }
+        if (!context.Parameter.MappedParameter.ParameterType.TryGetElementType(out var elementType)) { return; }
+        if (!elementType.GetMetadata().Has<LocatableAttribute>()) { return; }
+        if (elementType.TryGetQueryType(_domain, out var _)) { return; }
```

Comment:
this creates a dependency to orm feature. don't use orm or data access or related coding style extensions.